data/syntax/isocpp.xml

```diff
@@ -360,13 +360,13 @@
       </context>
 
       <context name="match identifier" attribute="Normal Text" lineEndContext="#pop">
-        <keyword attribute="Data Type" context="#stay" String="types" />
```

Comment:
The "Types" regex that matches `*_t` and this one show up as the same class, and I want to color them differently. In other words, I don't want `int` and `foo_t` to be the same color. Right now `foo_t<int>` is:

```html
<p><code class="sourceCode cpp"><span class="dt">foo_t</span>&lt;<span class="dt">int</span>&gt;</code></p>
```

and with this change `foo_t` stays a `dt` but `int` becomes a `kw`:

```html
<p><code class="sourceCode cpp"><span class="dt">foo_t</span><span class="op">&lt;</span><span class="kw">int</span><span class="op">&gt;</span></code></p>
```

Reply:
I just pushed 53ed30c9 to remove the unwanted parsing rules, specifically the `_t` rule.